src/index.html

```diff
@@ -431,68 +431,74 @@ <h3>Latest blog</h3>
       <div class="row">
         <div class="col-4">
           <div class="post-box">
-            <div class="cover">
-              <div class="details">
-                <div class="date">
-                  <i class="fas fa-calendar-alt"></i> 15 Jan 2016
-                </div>
-                <div class="comments">
-                  <i class="fas fa-comments"></i> 4 Comments
+            <a href="#" class="post-box-link">
+              <div class="cover">
+                <div class="details">
+                  <div class="date">
+                    <i class="fas fa-calendar-alt"></i> 15 Jan 2016
+                  </div>
+                  <div class="comments">
+                    <i class="fas fa-comments"></i> 4 Comments
+                  </div>
                 </div>
               </div>
-            </div>
-            <div class="post-info">
-              <h5>Products That Fight Static </h5>
-              <p>Aliquam dictum pellentesque nibh sit amet dapibus. Vivamus eget luctus nisi. Nullam euismod leo
-                vehicula, rutrum magna in, ornare enim vehicula, rutrum magna in, ornare enim.</p>
-              <div class="more">
-                <a href="#" class="btn-main">Read more</a>
+              <div class="post-info">
+                <h5>Products That Fight Static </h5>
```

Comment:
spróbujmy nie zostawiać elementów HTML bez atrybutu `class` pamiętasz z BEM? Pójdźmy w to

Reply:
Ok mogę poprawić, tyle że w kodzie jest pełno takich kwiatków. Mam poprawiać je wszystkie? Poza tym chyba też BEM nie jest tu stosowane, np. 
`<div class="cover">
              <div class="details">
                <div class="date">`

Reply:
nie musisz zmieniać tego w całym systemie, natomiast chciałbym, żebyś trzymała się tego czego się już nauczyłaś, wiec jak coś zmieniasz to trzymaj się konwencji BEM i będzie dobrze. Rozwiąż konflikty, które się pojawiły